src/query.c

```diff
@@ -980,8 +1017,18 @@ hy_query_filter(HyQuery q, int keyname, int cmp_type, const char *match)
 	HySack sack = query_sack(q);
```

Comment:
What if we delete the switch, use whats in default statement and save it as a string and convert it to Reldep during `hy_query_apply`?